docs/api/blink-utils.md

```diff
@@ -0,0 +1,26 @@
+# blinkUtils
+
+> A utility layer to interact with blink objects
+
+Process: [Renderer](../glossary.md#renderer-process)
+
+## Methods
+
+The `blinkUtils` module has the following methods:
+
+### `blinkUtils.getPathForFile(file)`
+
+* `file` File - A web [File](https://developer.mozilla.org/en-US/docs/Web/API/File) object.
+
+Returns `string` - The file system path that this `File` object points to. In the case where the object passed in is not a `File` object an exception is thrown. In the case where the File object passed in was constructed in JS and is not backed by a file on disk an empty string is returned.
```

Comment:
Is there anything we need to be doing to throw an exception when the object is not a `File`?